script.js

```diff
@@ -32,8 +36,81 @@ const strokeWeight = 1.5;
 const pointRadius = 8;
 const fillOpacity = 0.7;
 
+let initialMapZoom = 4;
+//setting the initial zoom settings
+if (IS_MOBILE){
+  initialMapZoom = 3;}
+else if (IS_DESKTOP){
+  initialMapZoom = 5;}
+
+//initial values, if not given by the url
+let lat = 40.67;
+let long = -97.23;
+let z = initialMapZoom;
+let statesOn = true;
+let citiesOn = true;
+let countiesOn = true;
```

Comment:
Because there are now so many variables relating to the default map configuration, it might be better to stash all of these default properties that relate to the map state in an Object. e.g.
```js
let mapConfig = {
  lat: 41.33,
  lng: -95.71,
  zoom: 4,
  states: true,
  cities: true,
  counties: true
};
```

then override them after parsing the URL and reference them e.g.

```js
const map = L.map("map", mapOptions).setView([mapConfig.lat, mapConfig.lng], mapConfig.zoom);
```